src/generator.csproj

```diff
@@ -11,6 +11,7 @@
     <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
     <BuildDir Condition="'$(BUILD_DIR)' != ''">$(BUILD_DIR)\</BuildDir>
     <BuildDir Condition="'$(BUILD_DIR)' == ''">build\</BuildDir>
+    <LangVersion>latest</LangVersion>
```

Comment:
So, I hope this does not hit us. I have seen issues in CI were latests was not 8.0 :/

Reply:
I think  is when we backported xcode11.4 to d16-6, but we are not backporting this, so I don't think it will hit us this time.